test/request_cache/plug_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule RequestCache.PlugTest do
+  @moduledoc false
+
+  use ExUnit.Case, async: true
+
+  defmodule FailingCacheModule do
+    def get(_cache_key) do
+      {:error, %{reason: :timeout}}
+    end
+  end
+
+  setup do
+    Application.put_env(:request_cache_plug, :request_cache_module, RequestCache.ConCacheStore)
```

Comment:
why do we put this in env? I think it is overwritten for all the tests by the other `put_env`

Reply:
Yeah it is, this is also the default so I'm not sure why setting it makes a difference

Reply:
Normally I would do this with `mox`, but that doesn't seem to be very popular. The outer `setup` resets the value for all the other tests, the `setup` local to the `describe` sets the failing module for just those tests.

Reply:
Removed